democracy/views/hearing.py

```diff
@@ -253,8 +253,9 @@ class HearingListSerializer(HearingSerializer):
 
     def get_fields(self):
         fields = super(HearingListSerializer, self).get_fields()
-        # Elide section and geo data when listing hearings; one can get to them via detail routes
+        # Elide section, contact person and geo data when listing hearings; one can get to them via detail routes
         fields.pop("sections")
+        fields.pop("contact_persons")
```

Comment:
Breaking change: I assume there are no API users who might use this field.

Reply:
Nope, the API is actually not published yet so the only user is the UI itself. Contact persons are not rendered in the event listing, so it actually makes no sense to fetch them from the database at this point.